lib/openfoodfacts.dart

```diff
@@ -589,7 +577,7 @@ class OpenFoodAPIClient {
     final OpenFoodFactsLanguage language,
   ) async {
     const String FIELD = 'environment_infocard';
-    final Uri uri = Settings.getUri(
+    final Uri uri = OpenFoodAPISettings().getUri(
```

Comment:
That can't work. We need to change the constructor of `OpenFoodAPISettings`.
Something like that:
```dart
OpenFoodAPISettings.fromQueryType(
    final QueryType queryType, {
    final OpenFoodFactsLanguage? language,
  })
     : this(
         scheme: _URI_SCHEME_HTTPS,
         host: queryType == QueryType.PROD ?
            (language != null ? 'world-${language.code}.openfoodfacts.org' : 'world.openfoodfacts.org')
            : (language != null ? 'world-${language.code}.openfoodfacts.net' : 'world.openfoodfacts.net'),
         robotoffHost: queryType == QueryType.PROD
             ? _URI_ROBOTOFF_HOST_PROD
             : _URI_ROBOTOFF_HOST_TEST,
       );
```